### PR TITLE
feat(proto): add completions to the package

### PR DESCRIPTION
**Pull request by liljaylj**
add completions for zsh, bash and fish shells. _i've not enough knowledge how to add completions for other shells (i.e. elvish and powershell)_

also, bumped package version

### CODE REVIEW

File: proto-bin/PKGBUILD [hunk 1]
```diff
@@ -15,10 +15,28 @@ url='https://github.com/moonrepo/proto'
 arch=('x86_64')
 depends=('git')
 source=("${pkgname}-${pkgver}.tar.xz::${url}/releases/download/v${pkgver}/proto_cli-${arch[0]}-unknown-linux-gnu.tar.xz")
-sha256sums=('bb1f6b01433ff982ceee8497a436ac2175fe73d6fcd1e7afab1ba897c1041419')
+b2sums=('f409590dca8d6b0dd1f366afc667b575150d478c4e2a78d3986778373db0d634e85abd220902e9bc80eec719bce210d58bd1706808459ab6bafdfa92fa62d3bd')
```
Comment (phnx47) [inline proto-bin/PKGBUILD:18]: Upstream use sha256, keep sha256sums, please.

https://github.com/moonrepo/proto/releases/download/v0.17.0/proto_cli-x86_64-unknown-linux-gnu.tar.xz.sha256

File: proto/PKGBUILD [hunk 2]
```diff
@@ -15,7 +15,7 @@ makedepends=('cargo')
 options=('!lto')
 _sha='975ef98'
```
Comment (phnx47) [inline proto/PKGBUILD:16]: Update `_sha` please, you can do it with `./update.sh proto` command.

**Review by phnx47 — CHANGES_REQUESTED** [review]
Thanks! I added some comments.

**Comment by phnx47** [comment]
Can you update `.SRCINFO`?

**Comment by phnx47** [comment]
@liljaylj Nice work! Let me test it, and then I will publish it to AUR.

**Comment by liljaylj** [comment]
@phnx47 thanks

would be awesome if you submit request to [add this packages to chaotic-aur repo](https://github.com/chaotic-aur/packages/issues/new/choose)

**Comment by phnx47** [comment]
@liljaylj I don't use `chaotic-aur`, looks like you can request packages yourself, but I suggest doing it when the `proto` will have more "votes" in AUR.

**Pull request merged**